offline-geocode/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,22 @@
+<resources>
+    <string name="app_name">Offline geocode</string>
+    <string name="geo_locate_error">Geocode failed on address</string>
+    <string name="location_not_foud">We are sorry, we couldn\'t find a location with the name - </string>
+    <!-- San Francisco tpk and locator paths -->
+    <string name="sandiego_tpk">/ArcGIS/samples/OfflineRouting/streetmap_SD.tpk</string>
+    <string name="sandiego_loc">/ArcGIS/samples/OfflineRouting/Geocoding/SanDiego_StreetAddress.loc</string>
```

Comment:
@manishk3189 The folder path on device does not match the folder path in the README.  Since this is the offline-geocoding sample use change the **/OfflineRouting/** to **/OfflineGeocoding/** for the device file path.